Chatto/Tests/ChatController/BaseChatViewControllerTestHelpers.swift

```diff
@@ -255,3 +264,13 @@ private final class FakeChatInputBarPresenter: BaseChatInputBarPresenterProtocol
 private final class FakeChatViewControllerViewModel: BaseChatViewControllerViewModelProtocol {
     var onDidUpdate: (() -> Void)?
 }
+
+private final class FakeKeyboardAdjuster: KeyboardUpdatesHandlerDelegate {
+    weak var viewController: BaseChatViewController?
+
+    func didAdjustBottomMargin(to margin: CGFloat, state: KeyboardState) {
+        guard let viewController = self.viewController else { return }
```

Comment:
It would be nice to use assertions in unexpected early returns of fake objects if they aren't covered with tests.